great_expectations_cloud/agent/agent.py

```diff
@@ -209,14 +208,25 @@ def _handle_event_as_thread_exit(
 
         # get results or errors from the thread
         error = future.exception()
+        # TODO Clean up ifs
         if error is None:
             result: ActionResult = future.result()
-            status = JobCompleted(
-                success=True,
-                created_resources=result.created_resources,
-            )
-            print(f"Completed job: {event_context.event.type} ({event_context.correlation_id})")
+
+            # TODO This result may have an UnknownEvent error
+            if result.event is UnknownEvent:
+                status = JobCompleted(
+                    success=False,
```

Comment:
The data model is below, how should I convey its an unknown event @anthonyburdi ?
```
class JobCompleted(AgentBaseModel):
    status: Literal["completed"] = "completed"
    success: bool
    created_resources: Sequence[CreatedResource] = []
    error_stack_trace: Union[str, None] = None
```